src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1205,10 +1205,10 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 		return NULL;
 #endif
 // FIXME: This limitation could be removed once everything here are supported by mini JIT on arm64
-// #ifdef TARGET_ARM64
-// 	if (!(cfg->compile_aot && cfg->full_aot && !cfg->interp))
-// 		return NULL;
-// #endif
+#ifdef TARGET_ARM64
```

Comment:
@fanyang-mono - can you confirm if all of these are supported to remove this check ?

Reply:
When all the API's are supported, we could remove this check.